### DISCUSSION TITLE
Лицензия "слетает" только после убийства персонажа

### DISCUSSION THREAD

**Discussion by octothorpteam** [octothorp-team/public-dobrograd#588] [Предложения]
## ![](https://forum.octothorp.team//user_avatar/forum.octothorp.team/hleb/32/2128_2.png) [Hleb](https://forum.octothorp.team/u/Hleb) ([#9316](https://forum.octothorp.team/p/9316))

- За
- Против

0голосов

Здравствуйте дорогие игроки Доброграда, ну и администрация тоже. Возможно уже многих достало делать одно и тоже - Сначало покупать оружие, а потом идти в мэрию покупать лицензию и потом понимаешь, что она пропала после выхода (из сервера) . И всё повторяется по кругу и это сильно раздражает. Я предлагаю сделать так - Лицензия пропадает после смерти персонажа.

**Comment by octothorpteam** [comment]
## ![](https://forum.octothorp.team//user_avatar/forum.octothorp.team/jusko/32/2001_2.png) [jusko](https://forum.octothorp.team/u/jusko) ([#9349](https://forum.octothorp.team/p/9349))


Добавь голосование

**Comment by octothorpteam** [comment]
## ![](https://forum.octothorp.team//user_avatar/forum.octothorp.team/hleb/32/2128_2.png) [Hleb](https://forum.octothorp.team/u/Hleb) ([#9353](https://forum.octothorp.team/p/9353))


Я тебя не понял, голосование на, что?

**Comment by octothorpteam** [comment]
## ![](https://forum.octothorp.team//user_avatar/forum.octothorp.team/jusko/32/2001_2.png) [jusko](https://forum.octothorp.team/u/jusko) ([#9360](https://forum.octothorp.team/p/9360))


На своё предложение. Раздел с предложениями требует добавления голосования, чтобы игроки могли проголосовать за или против него

**Comment by octothorpteam** [comment]
## ![](https://forum.octothorp.team//user_avatar/forum.octothorp.team/4cb/32/176_2.png) [4CB](https://forum.octothorp.team/u/4CB) ([#9384](https://forum.octothorp.team/p/9384))


В редактировании своей темы нажать на шестеренку и там будет “создать опрос”
![image](https://forum.octothorp.team/uploads/default/original/2X/f/f52f385b997f96f42fed9430ff3e2dd89e68bfaa.png)

**Comment by octothorpteam** [comment]
## ![](https://forum.octothorp.team//user_avatar/forum.octothorp.team/обжора/32/247_2.png) [Обжора](https://forum.octothorp.team/u/Обжора) ([#9443](https://forum.octothorp.team/p/9443))


Как мне кажется такое возможно только при появлении полноценной организации “Правительство”.

**Comment by octothorpteam** [comment]
## ![](https://forum.octothorp.team//user_avatar/forum.octothorp.team/belt/32/719_2.png) [BELT](https://forum.octothorp.team/u/BELT) ([#9546](https://forum.octothorp.team/p/9546))


# Привет ![:wave:](https://forum.octothorp.team/images/emoji/twitter/wave.png?v=10 ":wave:").

**Сразу перейду к разбору твоего предложения. В твоем предложение много недостатков и проблем. Добавление этой механики может частично убить такую функцию как “создания персонажа”. То-есть будет как на закрытом проекте Plattsburg, тебе будет разрешено создать только определенное количество персонажей. И только так можно сохранить лицензию. Да и к тому-же придется создавать организацию или систему правительства (Со слов человека сверху). Так-что я голосую против.**

**Comment by octothorpteam** [comment]
## ![](https://forum.octothorp.team//user_avatar/forum.octothorp.team/julianscandium/32/84_2.png) [julianscandium](https://forum.octothorp.team/u/julianscandium) ([#9595](https://forum.octothorp.team/p/9595))


Привет, [@Hleb](/u/hleb)
 Напомню, что ты можешь редактировать уже отправленные сообщения, вместо создания новых.

**Comment by LENFYS** [comment]
За
